configs/logging.go

```diff
@@ -0,0 +1,27 @@
+package configs
```

Comment:
Please add a description to these functions, check other functions in the same package as reference

Reply:
Done -> https://github.com/NethermindEth/sedge/pull/154/commits/90c1b86ffaa39c5b2228a65a0da873e5ff8062cc